src/test/java/seedu/address/logic/parser/EditCommandParserTest.java

```diff
@@ -166,36 +166,37 @@ public void parse_oneFieldSpecified_success() {
     }
 
     @Test
-    public void parse_multipleRepeatedFields_acceptsLast() {
+    public void parse_multipleRepeatedFields_failure() {
```

Comment:
I think I'm the one who suggested to use `_failure()` but I regret that now. We don't expect tests to fail. A better way to indicated the expected results is `_exceptionThrown()`

Reply:
Perhaps we can change `_failure()` to `_exceptionThrown` or some more appropriate terms to standardize the unit test formatting in a separate PR?

Reply:
Ah... so it is used elsewhere as well. In that case leave it be, for consistency.